android/app/build.gradle

```diff
@@ -66,4 +67,6 @@ flutter {
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    implementation platform('com.google.firebase:firebase-bom:31.1.1')
+    implementation 'com.google.firebase:firebase-analytics'
```

Comment:
remove firebase analytics dependency, it is not needed when you already have bom dependency